administrator/components/com_users/Model/UsersModel.php

```diff
@@ -96,6 +97,9 @@ protected function populateState($ordering = 'a.name', $direction = 'asc')
 
 		$this->setState('filter.excluded', $excluded);
 
+		$access = json_decode(base64_decode($app->input->get('access', '', 'BASE64')));
```

Comment:
This looks like something is wrong here.

Reply:
I modified it according to how it is done in ArticlesModel, is it ok now?